server/controllers/users.js

```diff
@@ -96,3 +96,46 @@ export function getAllUsers(req, res, next) {
       console.log('could not retrieve users', err);
     });
 }
+
+function toCamelCase(word) {
```

Comment:
Next time perhaps check lodash for helper methods like this one?

Reply:
Good call. I'll keep this in mind going forward.